packages/react-avatar/src/components/AvatarGroup/AvatarGroup.strings.tsx

```diff
@@ -0,0 +1,5 @@
+import type { AvatarGroupStrings } from './AvatarGroup.types';
+
+export const avatarGroupDefaultStrings: AvatarGroupStrings = {
+  tooltipContent: '{numOverflowedAvatars} more people.',
```

Comment:
_nit_: I feel like this message could be refined, but that can be done later.

Reply:
The original message was `Click to see {numOverflowedAvatars} more people.`, but Sarah said it would be better to use that one.

Reply:
Sounds good, thanks!